build/kl10/include.tcl

```diff
@@ -66,11 +71,12 @@ proc make_dskdmp {} {
     respond "?" "ASK\r"
     respond "HRIFLG=" "N\r"
     respond "BOOTSW=" "N\r"
-    respond "RP06P=" "N\r"
-    respond "RP07P=" "N\r"
+    respond "R11R6P=" "N\r"
+    respond "R11R7P=" "N\r"
     respond "RM03P=" "N\r"
     respond "RM80P=" "N\r"
     respond "RH10P=" "Y\r"
+    respond "R10R6P=" "Y\r"
```

Comment:
RP10R6P=Y seems to me DSKDMP is assembled for RH10 and RP06, right?

Reply:
Whoops.  Must be that we're not using this now -- since the build worked fine and the resulting system works.  I'll fix that.

Reply:
I just fixed this (changed the Y to an N).  I'm going to retest the KL build locally.

Reply:
I got past the build of DSKDMP in my local build. Our change to use DDT rather than DSKDMP now means that we don't test out the generated DSKDMP. Maybe we should load DSKDMP from DDT during the build one time and use it just to test out that it works.

Reply:
Yes, good idea!